frontends/concrete-python/concrete/fhe/compilation/configuration.py

```diff
@@ -46,6 +46,32 @@ def parse(cls, string: str) -> "ParameterSelectionStrategy":
         raise ValueError(message)
 
 
+class MultiParamStrategy(str, Enum):
```

Comment:
Could you rename it to `MultiParameterStrategy`, we used the long name in the other places, it's better to be consistent.

Reply:
Done